docs/docs/webchat/webchat-button-disable.md

```diff
@@ -0,0 +1,89 @@
+---
+id: webchat-button-disable
+title: Enabling and Disabling Buttons
+---
+
+This functionality allows you to disable a button once the user has clicked on it. Thanks to this visual information, the user can better understand the logical flow and navigate more easily.
+
+To add this property by default, you must set `autodisable: true` in the webchat `index.js`.
+The button style properties must be set as well.
+
+**index.js**
+
+```javascript
+export const webchat = {
+  theme: {
+    style: {
+      width: 500,
+    },
+    button: {
+      disabledstyle: {
+        opacity: 0.5,
+        cursor: 'auto',
+        pointerEvents: 'none',
+        backgroundColor: 'green',
+      },
+      autodisable: true,
+    },
+  },
+}
+```
+
+
+## Example with a button
+
+In `button.js`, if you add:
```

Comment:
You can also overwrite this behavior in a more specific level by defining these props in a specific `Button`.